src/Test/Utilities/Portable/CompilerTraitAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Unfortunately we need both.  The `TraitDiscoverer` attribute specifies the assembly where the actual discoverer object lives.  The name is different between our portable and desktop version.